charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.4.9
+version: 1.4.10
```

Comment:
IMHO version 1.5.0 would be appropriate as we are adding a new registry.

Reply:
done